services/blockchain-indexer/shared/utils/account.js

```diff
@@ -20,29 +20,35 @@ const {
 } = require('@liskhq/lisk-cryptography');
 
 const {
+	Logger,
 	MySQL: { getTableInstance },
 } = require('lisk-service-framework');
 
 const accountsTableSchema = require('../database/schema/accounts');
 const config = require('../../config');
 
+const logger = Logger();
+
 const MYSQL_ENDPOINT = config.endpoints.mysql;
 
 const getAccountsTable = () => getTableInstance(accountsTableSchema, MYSQL_ENDPOINT);
 
 const getLisk32AddressFromPublicKey = publicKey => getLisk32AddressFromPublicKeyHelper(Buffer.from(publicKey, 'hex'));
 
 const updateAccountPublicKey = async (publicKey) => {
```

Comment:
Please remove the `try...catch` here too and as discussed offline, please create a new job queue that takes care of updating the account information. The job should automatically retry if the action fails.